src/components/teaser/teaser.ts

```diff
@@ -19,30 +21,26 @@ import {
 } from '../core/interfaces';
 import type { TitleLevel } from '../title';
 import '../title';
+import '../chip';
 
 import style from './teaser.scss?lit&inline';
 
 /**
  * It displays an interactive image with caption.
  *
- * @slot image - Slot used to render the image
- * @slot title - Slot used to render the title
- * @slot description - Slot used to render the description
+ * @slot image - Slot used to render the image.
+ * @slot chip - Slot used to render the sbb-chip label.
+ * @slot title - Slot used to render the title.
+ * @slot description - Slot used to render the description.
  */
 @customElement('sbb-teaser')
 export class SbbTeaser extends LitElement implements LinkProperties {
   public static override styles: CSSResultGroup = style;
 
-  /**
-   * Teaser variant -
-   * when this is true the text-content will be under the image
-   * otherwise it will be displayed next to the image.
-   */
-  @property({ attribute: 'is-stacked', reflect: true, type: Boolean }) public isStacked: boolean;
+  /** Teaser variant - define the position and the alignment of the text block. */
+  @property({ reflect: true }) public alignment: 'end-centered' | 'end-top' | 'bottom';
```

Comment:
In Figma it's bottom, but I think we normally don't want to use the "old" terms, but maybe I am wrong :D

Reply:
Actually in this case "end" means "right"
For example "end-centered": text at the right of the image, vertically centered
Instead bottom... is bottom 😄 
Perhaps I could change the "end-centered" and "end-top" into "right-centered" and "right-top"?

Reply:
Maybe `block-end` and `inline-end-centered` and `inline-end-start` would be the right terms?

Reply:
Or to be more precise: `inline-end-block-centered` and `inline-end-block-start` and `block-end`

Reply:
I don't have the right answer but in the toast component we use `top | bottom` for vertical alignment and `start | end` for horizontal

Reply:
Everyone has the right to answer! :D
I was also not happy with the terms there :P. Because depending browser settings / locale it would not be bottom, but top. But on the other hand, I know that our library probably never will be consumed or used in this situations...

Reply:
I know of the `right-to-left` mode. But never heard of a `bottom-to-top` mode 😵‍💫 
Btw, I copied angular-material for the [toast](https://material.angular.io/components/snack-bar/api#type_aliases)

Reply:
My two cents: `inline-centered` / `inline-end` / `block-end`.

Reply:
what has influence is the css writing-mode (https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode?retiredLocale=de). I know it's a rare case, I just think for the CSS we do big effort to always use those properties but why not use them as well in component property to be consistent?

Reply:
Another idea might be to use `above | below` for vertical and `before | after` for horizontal

Reply:
I would personally go with `inline-end-block-centered` + `inline-end-block-start` + `block-end`
they are the "most precise" and are still correct with different writing modes

Reply:
Let's discuss this in refinement on monday :)

Reply:
after discussion: align-text: after | after-centered | below

Reply:
edited and pushed